plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -217,6 +217,10 @@ class WC_Install {
 			'wc_update_700_remove_download_log_fk',
 			'wc_update_700_remove_recommended_marketing_plugins_transient',
 		),
+		'7.2.0' => array(
```

Comment:
Does this need to be 7.2.1?

Reply:
As discussed this is handled in the follow up cherry pick we'll do.